src/core/tooltip_text.py

```diff
@@ -0,0 +1,14 @@
+RANDOM = 'Check this box to randomly generate an safety car event.'
+MAXIMUM_OCC = 'The maximum number of times a safety car event could potentially occur (not guaranteed).'
+PROBABILITY = 'The likelihood of a safety car being deployed. A value of 1 indicates a 100% chance of a safety car being deployed, while a value of 0 means there is no probability of a safety car.'
```

Comment:
A clarification: it's the probability of one being deployed. Even if the maximum is set to, say, 5, this percentage here is still the probability of a single one being deployed randomly, not the chance of all 5.